### PR TITLE
Fix "pandas.Dataframe" to "pandas.DataFrame" in pygmt/datasets/samples.py

### LINKED ISSUE #1123: Fix "pandas.Dataframe" to "pandas.DataFrame" in pygmt/datasets/samples.py
In [pygmt/datasets/samples.py](https://github.com/GenericMappingTools/pygmt/blob/master/pygmt/datasets/samples.py), sometimes we use "pandas.Dataframe", but the correct name should be "pandas.DataFrame" (capital **F**).

### PR DESCRIPTION
**Pull request by MichaeINeumann**
**Description of proposed changes**

Replace (Match case and WholeWord) "pandas.Dataframe" to "pandas.DataFrame"

Fixes #1123 

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

File: pygmt/datasets/samples.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ def load_japan_quakes():
 
     Returns
     -------
-    data :  pandas.Dataframe
+    data :  pandas.DataFrame
```
Comment (seisman) [inline pygmt/datasets/samples.py:21]:
```suggestion
    data : pandas.DataFrame
```

File: pygmt/datasets/samples.py [hunk 2]
```diff
@@ -49,7 +49,7 @@ def load_ocean_ridge_points():
...
```
Comment (seisman) [inline pygmt/datasets/samples.py:52]:
```suggestion
    data : pandas.DataFrame
```

File: pygmt/datasets/samples.py [hunk 3]
```diff
@@ -72,7 +72,7 @@ def load_sample_bathymetry():
...
```
Comment (seisman) [inline pygmt/datasets/samples.py:75]:
```suggestion
    data : pandas.DataFrame
```

**Review by seisman — COMMENTED** [review]
@MichaeINeumann Thanks for your quick fix!

Just a few tiny suggestions to remove the extra whitespace before `pandas.DataFrame`.

**Comment by MichaeINeumann** [comment]
"Just a few tiny suggestions to remove the extra whitespace before pandas.DataFrame."
Sorry, you did this changes? It's my first time at gitHub. Do you expected enything else from me? THX

**Comment by seisman** [comment]
> "Just a few tiny suggestions to remove the extra whitespace before pandas.DataFrame."
> Sorry, you did this changes? It's my first time at gitHub. Do you expected enything else from me? THX

Sorry, I shouldn't assume any git or github knowledge for new contributors.

As you can see, I just made three *suggestions* for this PR, but they're **NOT** committed to this PR yet:

image

If you agree with the *suggested changes*, you can click the "Commit suggestion" button to make the changes. Each time you click the button, the suggested changes will be committed to your branch.

Alternatively, you can go to the "Files changes" tab:

image

and click the "Add the suggestion to batch" button, 

image

and then click the "Commit suggestion" button on the top, which will make all changes in a single commit:
image

**Comment by seisman** [comment]
Great! Just one more tip, when you make some changes and submit a PR, please choose "Create a new branch for this commit and start a pull request" and click on the "Propose changes" button to open a pull request.

image

It would be better than choosing "commit directly to the master branch".

**Review by seisman — APPROVED** [review]

**Pull request merged**